client/src/App.js

```diff
@@ -0,0 +1,46 @@
+import Header from './components/Header';
+import Home from './pages/Home';
+import Navigation from './components/Navigation';
+import { Route, Switch } from 'react-router-dom';
+import { useState } from 'react';
```

Comment:
```suggestion
import { useState } from 'react';
import { Route, Switch } from 'react-router-dom';
import Home from './pages/Home';
import Header from './components/Header';
import Navigation from './components/Navigation';
```

Usually we structure the imports from outside in: 1) external libraries 2) your own pages and components